fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/expressions/Comparisons.java

```diff
@@ -2557,6 +2595,17 @@ public Comparison translateCorrelations(@Nonnull final TranslationMap translatio
             }
         }
 
+        @Nonnull
+        @Override
+        @SuppressWarnings("PMD.CompareObjectsWithEquals")
+        public Comparison translateValue(@Nonnull final UnaryOperator<Value> valueTranslator) {
```

Comment:
I think this can then be done by the default, no?

Reply:
No, `inner` is special to `MultiColumnComparison`.